web/virtual_classroom_views.py

```
@@ -0,0 +1,785 @@
+    # Create seats if they don't exist
+    if created:
+        for row in range(classroom.rows):
+            for col in range(classroom.columns):
+                VirtualSeat.objects.create(
+                    classroom=classroom,
+                    row=row,
+                    column=col,
+                )
+    
+    return redirect('virtual_classroom', session_id=session.id)
+    SessionAttendance,
```

Ruff (0.8.2)

Found useless expression. Either assign it to a variable or remove it.

(B018)